modules/augment/test/augmenter_images.py

```diff
@@ -0,0 +1,35 @@
+try:
+    import cv2 as cv
+except ImportError:
+    raise ImportError('Can\'t find OpenCV Python module. If you\'ve built it from sources without installation, '
+                      'configure environment variable PYTHONPATH to "opencv_build_dir/lib" directory (with "python3" subdirectory if required)')
```

Comment:
Kind of redundant exception message: import cv2 already produces enough information